rpc/block_common.h

```diff
@@ -29,7 +29,7 @@
 
 # define   GB_TGCLI_GLFS_PATH   "/backstores/user:glfs"
 # define   GB_TGCLI_GLFS        "targetcli " GB_TGCLI_GLFS_PATH
-# define   GB_TGCLI_CHECK       GB_TGCLI_GLFS " ls | grep ' %s ' | grep '/%s ' > " DEVNULLPATH
+# define   GB_TGCLI_CHECK       GB_TGCLI_GLFS " ls | grep ' %s ' | grep '/%s' > " DEVNULLPATH
```

Comment:
We don't need this patch, squashing it to the next would be enough.

Reply:
No we need this.

Just to see what if it likes:
`
  | | o- block102 ... [repvol@localhost/block-store/ced05dab-f685-4fdd-b257-1a86fe89702a;tcmur_cmd_time_out=44 (100.0MiB) activated]`

Reply:
Sorry, what I mean was that, we don't need a split patch, we can squash it to the next.